File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Dias/107.pl
print "Escolha a opcao do menu referente ao periodo desejado:\n";
print "1 - Verao\n2 - Outono\n3 - Inverno\n4 - Primavera\n";
$opcao = <STDIN>;
chomp ($opcao);
if ($opcao == 1)
{
	print "Verao\n";
}
elsif ($opcao == 2)
{
	print "Outono\n";
}
elsif ($opcao == 3)
{
	print "Inverno\n";
}
elsif ($opcao == 4)
{
	print "Primavera\n";
}
else
{
	print "Insira entre 1 e 4\n";
}

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Dias/107.rb
puts "Escolha a opcao do menu referente ao periodo desejado:"
puts "1 - Verao\n2 - Outono\n3 - Inverno\n4 - Primavera"
opcao = gets.chomp
if opcao.to_i == 1
  puts "Verao"
elsif opcao.to_i == 2
  puts "Outono"
elsif opcao.to_i == 3
  puts "Inverno"
elsif opcao.to_i == 4
  puts "Primavera"
else
  puts "Insira entre 1 e 4"
end